src/main/java/io/kestra/plugin/kestra/executions/Resume.java

```diff
@@ -0,0 +1,81 @@
+package io.kestra.plugin.kestra.executions;
+
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.kestra.AbstractKestraTask;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.util.Map;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Resume a paused execution",
+    description = "This task uses the Kestra API to resume an execution that is in a PAUSED state."
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Resume an execution",
+            code = {
+                "executionId: \"{{ trigger.executionId }}\""
+            }
+        )
+    }
+)
+public class Resume extends AbstractKestraTask implements RunnableTask<Resume.Output> {
+
+    @Schema(title = "The Execution ID to resume")
+    private Property<String> executionId;
+
+    @Schema(title = "Map of inputs to send to the paused execution")
+    private Property<Map<String, Object>> inputs;
+
+    @Override
+    public Output run(RunContext runContext) throws Exception {
+        // 1. Render Execution ID
+        String rExecutionId = runContext.render(this.executionId).as(String.class).orElseThrow();
```

Comment:
It should default to the current execution if executionId is not set as in the original task.
Please also specify this inside the property documentation.

You can get the current executionId via `(String) new HashMap<>((Map<String, Object>) runContext.getVariables().get("execution")).get("id")`